frontend/src/app/services/event-list.ts

```diff
@@ -60,8 +80,33 @@ export function updateEventData(eventInput: EventInput) {
   } else {
     eventData.tooManyDeathsDays = 0;
   }
+
+  eventData.showAntivaxEvent = eventInput.date > '2020-01-10'
+    && isEventTriggerActive(eventInput, ANTIVAX_WITH_CAMPAIGN_TRIGGER)
+    && isEventTriggerActive(eventInput, ANTIVAX_WITHOUT_CAMPAIGN_TRIGGER)
+    && probability(0.02);
 }
 
+const antivaxEvents = [
+  {
+    title: 'Toto jsou fakta: vakcína proti koronaviru je tvořena fragmenty tkání z potracených lidských plodů!',
+    text: 'Ve společnosti se šíří hoax o tom, že vakcína obsahuje látky z nenarozených dětí.',
+    help: 'Rychlost vakcinace se snižuje.',
+    mitigations: [
+      {label: 'Ok', vaccinationPerDay: -0.0002, timeout: 1},
+    ],
+  },
+];
+
+const antivaxEventsWithCampaign = antivaxEvents.map(e => ({
+  ...e,
+  help: 'Očkovací kampaň přestala fungovat.',
+  mitigations: [
+    {label: 'Ok', id: VACCINATION_CAMPAIGN_ID, timeout: 0},
```

Comment:
`VACCINATION_CAMPAIGN_ID` is probably a remainder of some experiments, there's a regular item with this ID. Reusing ids is possible and risky, hence. Am I correct this testing event should also have `vaccinationPerDay` defined?